plugin/documents.py

```diff
@@ -483,8 +483,16 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'region.yellowish lightbulb.lsp'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
-            suffix = 's' if action_count > 1 else ''
-            code_actions_link = make_command_link('lsp_code_actions', '{} code action{}'.format(action_count, suffix))
+            if action_count > 1:
+                title = '{} code actions'.format(action_count)
+            else:
+                title = ''
+                for actions in responses.values():
+                    title = actions[0].get('title')
```

Comment:
Mypy doesn't like it. Maybe:

```suggestion
                    title = actions[0].get('title', 'Code action')
```